README.md

```diff
@@ -70,6 +70,7 @@ Sometimes, having files in a spreadsheet format is more useful for analysts and
 
 **Other Trackers/Dashboards**
 
+- [Covid19-Tracker-App](https://cutt.ly/covid19trackerapp) (by [@HarshitChauhan](https://github.com/HarshitChauhan))
```

Comment:
Could you move it to the end of the stack, just to be fair to others?

Also, would be great if you mention the covid19india.org source in your dashboard too. (Not mandatory though :) )

Reply:
i have specially mentioned in my dashboard in the sidebar there is cup gif if you click on it , it will show

Reply:
ok sure.